tests/unit/lms/services/event_test.py

```diff
@@ -60,6 +60,26 @@ def test_insert_event_no_data(self, svc, db_session):
         )
         assert not db_session.query(EventData).one_or_none()
 
+    def test_insert_event_type(self, svc, db_session):
+        type_query = db_session.query(EventType).filter_by(
+            type=EventType.Type.CONFIGURED_LAUNCH
+        )
+
+        # Insert the event type for the first time
+        svc.insert_event(
+            BaseEvent(request=sentinel.request, type=EventType.Type.CONFIGURED_LAUNCH)
+        )
+        # It will create a new type
+        event_type = type_query.one()
+        # Clear the @lru_cache of the private method
+        svc._get_type_pk.cache_clear()  # pylint:disable=protected-access
+        # Insert the event type again
+        svc.insert_event(
+            BaseEvent(request=sentinel.request, type=EventType.Type.CONFIGURED_LAUNCH)
+        )
+        # It will create a new type
```

Comment:
This comment doesn't seem quite right? Are we checking we get the same one back?

The arrange, act, assert here is a bit unclear

Reply:
Updated the comment and added a bit of white space.